app/services/metrics_service.rb

```diff
@@ -1,13 +1,14 @@
 require 'gds_api/content_data_api'
 
 class MetricsService
-  def fetch(base_path:, from:, to:, metrics:)
-    url = api.request_url(base_path: base_path, from: from, to: to, metrics: metrics)
+  DEFAULT_METRICS = %w[pageviews unique_pageviews number_of_internal_searches satisfaction_score].freeze
+  def fetch(base_path:, from:, to:)
```

Comment:
The method `fetch` does not reveal very well what it does. I know this is not something that you have introduced, but this might be a good opportunity to rename it. Effectively it just retrieves AggregatedMetrics, so it would rename it to better represent what it does.

This might be a good opportunity to split this Service into two:

1. FindTimeSeries
2. FindAggregatedValues

Reply:
I've renamed the method to `fetch_aggregated_data`.

The only reason i can see to split this class into 2 at the moment apart from it has 2 methods.

I'm not sure this is a good enough reason to split it.